docs/docs/concepts/architecture/solution.md

```diff
@@ -1,67 +1,60 @@
 ---
-title: ZITADEL's Deployment Architecture
+title: Zitadel's Deployment Architecture
 sidebar_label: Deployment Architecture
 ---
 
 ## High Availability
 
-ZITADEL can be run as high available system with ease. 
+Zitadel can be run as high available system with ease. 
 Since the storage layer takes the heavy lifting of making sure that data in synched across, server, data centers or regions.
 
-Depending on your projects needs our general recommendation is to run ZITADEL and ZITADELs storage layer across multiple availability zones in the same region or if you need higher guarantees run the storage layer across multiple regions.
-Consult the [CockroachDB documentation](https://www.cockroachlabs.com/docs/) for more details or use the [CockroachCloud Service](https://www.cockroachlabs.com/docs/cockroachcloud/create-an-account.html)
-Alternatively you can run ZITADEL also with Postgres which is [Enterprise Supported](/docs/support/software-release-cycles-support#partially-supported).
-Make sure to read our [Production Guide](/self-hosting/manage/production#prefer-postgresql) before you decide to use it.
+Depending on your projects needs our general recommendation is to run Zitadel across multiple availability zones in the same region or across multiple regions.
+Alternatively you can run Zitadel with Postgres which is [Enterprise Supported](/docs/support/software-release-cycles-support#partially-supported).
```

Comment:
alternatively to what? and also as Postgres is our only option, I guess we can omit the Enterprise Supported part

Reply:
removed